### PR TITLE
Update k8s v1.20.13

### PR DESCRIPTION
**Pull request by briandowns**
#### Types of Changes ####

Updates rke2 1.20 branch to latest patch.

#### Linked Issues ####

#2106

### CODE REVIEW

**Review by galal-hussein — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -93,8 +93,8 @@ RUN rm -vf /charts/*.sh /charts/*.md
 # This image includes any host level programs that we might need. All binaries
 # must be placed in bin/ of the file image and subdirectories of bin/ will be flattened during installation.
 # This means bin/foo/bar will become bin/bar when rke2 installs this to the host
-FROM rancher/k3s:v1.20.11-k3s2 AS k3s
-FROM rancher/hardened-kubernetes:v1.20.12-rke2r1-build20211028 AS kubernetes
+FROM rancher/k3s:v1.20.13-k3s1 AS k3s
+FROM rancher/hardened-kubernetes:v1.20.13-rke2r1-build20211119 AS kubernetes
 FROM rancher/hardened-containerd:v1.4.11-k3s1-build20211004 AS containerd
 FROM rancher/hardened-crictl:v1.19.0-build20210223 AS crictl
 FROM rancher/hardened-runc:v1.0.2-build20210915 AS runc
```
Comment (brandond) [inline Dockerfile:100]: Updated builds of these that have been rebuilt with the new golang version are also available and should be used for this release. Same with the etcd image (`v3.4.13-k3s3-build20211118`)
```suggestion
FROM rancher/hardened-containerd:v1.4.12-k3s1-build20211118 AS containerd
FROM rancher/hardened-crictl:v1.20.0-build20211118 AS crictl
FROM rancher/hardened-runc:v1.0.2-build20211118 AS runc
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by Oats87 — APPROVED** [review]

**Review by brandond — CHANGES_REQUESTED** [review]
etcd tag still needs bumping, at
* https://github.com/rancher/rke2/blob/release-1.20/scripts/build-binary#L27
* https://github.com/rancher/rke2/blob/release-1.20/scripts/build-images#L19

**Comment by briandowns** [comment]
Updated.

File: scripts/version.sh [hunk 2]
```diff
@@ -31,9 +31,9 @@ REVISION=$(git rev-parse HEAD)$(if ! git diff --no-ext-diff --quiet --exit-code;
...
 # hardcode versions unless set specifically
-KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.20.12}
-KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.20.12-rke2r1-build20211028}
-ETCD_VERSION=${ETCD_VERSION:-v3.4.13-k3s1}
+KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.20.13}
+KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.20.13-rke2r1-build20211119}
+ETCD_VERSION=${ETCD_VERSION:-v3.4.13-k3s3}
 PAUSE_VERSION=${PAUSE_VERSION:-3.2}
 CCM_VERSION=${CCM_VERSION:-v0.0.3-build20211118}
 IMAGE_BUILD_VERSION=${IMAGE_BUILD_VERSION:-build20210223}
```
Comment (brandond) [inline scripts/version.sh:39]: I think this needs to be bumped too, same as in 1.21
```suggestion
IMAGE_BUILD_VERSION=${IMAGE_BUILD_VERSION:-build20211118}
```

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/version.sh (hunk 2 above)
Comment (briandowns) [reply scripts/version.sh]: Resolved.

**Review by brandond — APPROVED** [review]

**Pull request merged**